spec/pulse/message_spec.cr

```diff
@@ -1,6 +1,31 @@
 require "../spec_helper"
+ROUTE_BASE          = "/api/portal/v1/"
+MOCK_INSTANCE_ID    = "mock-id"
+MOCK_INSTANCE_EMAIL = "test@place.tech"
```

Comment:
Instead of redeclaring these constants, make a file that gets required by `spec_helper.cr`

Reply:
No worries, all done.